app/views/user_mailer/send_otp.html.erb

```diff
@@ -0,0 +1 @@
+Your OTP for <%= link_to 'Violet', root_url %> is: <%= @user.current_otp %>
```

Comment:
@ajaystha42 why are you hard coding Violet here? App name is different depending on the subdomain and `APP_HOST`

Reply:
@donrestarone  is this ok? Your OTP for [root@lvh.me:5250]((http://root.lvh.me:5250/)) is: